js/validate.js

```diff
@@ -0,0 +1,45 @@
+const form = document.querySelector('#my-form');
+const emailInput = document.querySelector('#email');
+const errorMessage = document.querySelector('#error-message');
+
+async function handleSubmit(event) {
+  event.preventDefault();
+
+  const isEmailValid = emailInput.value === emailInput.value.toLowerCase();
+
+  if (isEmailValid) {
+    errorMessage.textContent = '';
+
+    console.log('Form submitted successfully!');
+  } else {
+    errorMessage.textContent = 'Email must be in lowercase.';
+  }
+
+  const status = document.getElementById('my-form-status');
+  const data = new FormData(event.target);
+  fetch(event.target.action, {
+    method: form.method,
+    body: data,
+    headers: {
+      Accept: 'application/json',
+    },
+  })
+    .then((response) => {
+      if (response.ok) {
+        status.innerHTML = 'Thanks for your submission!';
+        form.reset();
+      } else {
+        response.json().then((data) => {
```

Comment:
also based on your logic here i dont think this is what you want in all cases,,you have an error here and it still says thanks for your submission whereas the form shouldnt submit at all if there is an error

![image](https://github.com/moshoodabdullahi/portfolio/assets/105943931/6e7983bc-21d9-4a8a-91ee-4d550950c7c1)

Reply:
Done